systems/trajectory_optimization/dircon_util.cc

```diff
@@ -24,8 +24,9 @@ void checkConstraints(const MathematicalProgram* prog,
     bool isSatisfied = (y.array() >= c->lower_bound().array() - tol).all() &&
                        (y.array() <= c->upper_bound().array() + tol).all();
     if (!isSatisfied) {
-      std::cout << "Constraint violation: " << c->get_description() << std::endl;
-      MatrixXd tmp(y.size(),3);
+      std::cout << "Constraint violation: "
+          << c->get_description() << std::endl;
```

Comment:
I **think** the `<<` should go at the end of the line prior. It's easier to read that way.